config/environments/production.rb

```diff
@@ -32,7 +32,7 @@
   # `config.assets.precompile` and `config.assets.version` have moved to config/initializers/assets.rb
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = 'http://assets.example.com'
+  config.action_controller.asset_host = 'https://navody.digital'
```

Comment:
Url musia byt absolutne v mailoch, ak by niekto pouzil ActionView helper ako napriklad `image_tag`.

Reply:
Toto by som asi zobral z ENV, lebo mame aj staging env s url staging.navody.digital

Reply:
Hotovo :)

Inac ja by so presunul `app.yml` do `secrets.yml`, je to trosku jednoduchsie ziskat config potom.